system/jlib/jsocket.cpp

```diff
@@ -2060,16 +2077,21 @@ void CSocket::read(void* buf, size32_t min_size, size32_t max_size, size32_t &si
         }
         size_read += rc;
     } while (size_read < min_size);
+
+    cycle_t elapsedCycles = get_cycles_now()-startcycles;
     STATS.reads++;
     STATS.readsize += size_read;
-    STATS.readtime+=usTick()-startt;
+    STATS.readtimecycles+= elapsedCycles;
```

Comment:
time+=u / cycles+= elapsed, etc should use consistent spacing of = / += etc. but NBD with me.